### PR TITLE
[bitnami/*] Update CI *-values.yaml files

### PR DESCRIPTION
**Pull request by andresbono**
**Description of the change**

- Execute the GitHub action to install the chart with default values in all the charts where is possible.
- Fix charts with failing installations.

**Possible drawbacks**

None.

**Applicable issues**

  - Related to https://github.com/bitnami/charts/pull/4487

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [ ] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

File: bitnami/keycloak/ci/ct-values.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+service:
+  type: ClusterIP
```
Comment (juan131) [inline bitnami/keycloak/ci/ct-values.yaml:2]: What happens with other available YAMLs in Keycloak? I mean:
- values-ha.yaml
- values-hpa-pdb.yaml
- values-metrics-and-ingress.yaml
Same question applies to the rest of charts

Comment (andresbono) [reply bitnami/keycloak/ci/ct-values.yaml:2]: We thought on renaming all the `ci/*.yaml` files to `ci/*-values.yaml`. However, it actually doesn't make sense. Many of the files are trying to test production settings and those can't be applied in a kind cluster.

**Review by juan131 — APPROVED** [review]
LGTM

File: bitnami/keycloak/ci/ct-values.yaml (hunk 1 above)
Comment (juan131) [reply bitnami/keycloak/ci/ct-values.yaml:2]: It makes sense

**Pull request merged**